doc/classes/TabContainer.xml

```diff
@@ -258,5 +258,8 @@
 		<theme_item name="tab_unselected" data_type="style" type="StyleBox">
 			The style of the other, unselected tabs.
 		</theme_item>
+		<theme_item name="tabbar_background" data_type="style" type="StyleBox">
```

Comment:
The snakecase-ification of `TabBar` would be `tar_bar` so this should maybe be `tab_bar_background`.

Reply:
Yeah, but it looks so awkward to me 😅

Reply:
I will do a pass on all theme items this week. I can work on consistency there, if this is imporant.